### PR TITLE
I have read the CLA Document and I hereby sign the CLA

### PR DESCRIPTION
**Pull request by 7ttp**
I have read the CLA Document and I hereby sign the CLA

### CODE REVIEW

File: domains/itsvaibhav.json [hunk 1]
```diff
@@ -0,0 +1,9 @@
+{
+    "owner": {
+        "username": "Vaibhav1337",
+        "email": "vaibhavtripathi1337@gmail.com"
+    },
+    "record": {
+        "TXT": "vc-domain-verify=itsvaibhav.is-a.dev,8b5014bc53898eae14fb"
```
Comment (Stef-00012) [inline domains/itsvaibhav.json:7]: vercel is not supported yet (might be soon)

File: domains/itzvaibhav .json [hunk 2]
```diff
@@ -0,0 +1,9 @@
+{
+    "owner": {
+        "username": "Vaibhav1337",
+        "email": "vaibhavtripathi1337@gmail.com"
+    },
+    "record": {
+        "TXT": "vc-domain-verify=itzvaibhav.is-a.dev,c1d24961d3d2e30cad09"
```
Comment (Stef-00012) [inline domains/itzvaibhav .json:7]: vercel is not supported yet (might be soon)

File: domains/itzvaibhav.json [hunk 3]
```diff
@@ -0,0 +1,9 @@
+{
+    "owner": {
+        "username": "Vaibhav1337",
+        "email": "vaibhavtripathi1337@gmail.com"
+    },
+    "record": {
+        "TXT": "vc-domain-verify=itzvaibhav.is-a.dev,c1d24961d3d2e30cad09"
```
Comment (Stef-00012) [inline domains/itzvaibhav.json:7]: vercel is not supported yet (might be soon)

**Review by Stef-DP — CHANGES_REQUESTED** [review]

File: domains/itsvaibhav.json (hunk 1 above)
Comment (Vaibhav1337) [reply domains/itsvaibhav.json:7]: is netlify supported?, if not can u pls suggest few of them

**Pull request closed without merging**